src/components/NavBar.vue

```diff
@@ -221,6 +221,22 @@ export default {
     @apply mx-1;
 }
 
+#search-btn:hover {
+    @apply bg-red-400;
+}
+
+.dark #search-btn:hover {
+    @apply bg-dark-100;
+}
+
+.nav-link {
+    @apply hover:font-bold hover:text-red-500;
+}
+
+.dark .nav-link {
+    @apply hover:font-bold hover:text-red-400;
```

Comment:
We might need to reconsider this, we get some layout shifting when hovering on the text due to the bold text.